libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -78,6 +78,8 @@ class AP_BattMonitor
         float       resistance;                // resistance, in Ohms, calculated by comparing resting voltage vs in flight voltage
         BatteryFailsafe failsafe;              // stage failsafe the battery is in
         bool        healthy;                   // battery monitor is communicating correctly
+        float       cell_min;                  // lowest smart battery cell voltage
```

Comment:
I'd keep this out of state, it's increasing the general memory footprint for all backends, and assuming a 10Hz SMBus smart battery (which is what we currently do) this isn't actually helping you save processing time. It's also forcing a more complicated getter relationship then you really need.

Reply:
Done